core/tests/sharing_tests.rs

```diff
@@ -299,9 +299,9 @@ fn write_document_in_rejected_shared_folder_in_share_folder() {
 
     assert_eq!(cores[1].read_document(document.id).unwrap(), b"document content by sharee");
     cores[1].sync(None).unwrap();
-    cores[1].get_file_by_id(document.id).unwrap_err();
+    cores[1].get_file_by_id(document.id).unwrap();
     cores[0].sync(None).unwrap();
-    assert_eq!(cores[0].read_document(document.id).unwrap(), b"document content by sharer");
+    assert_eq!(cores[0].read_document(document.id).unwrap(), b"document content by sharee");
```

Comment:
this test just had the wrong behavior before; the sharee should still have access to the file because it's in two shared folders and the pending share was only deleted for one of them